build-tools/packages/build-tools/src/common/fluidUtils.ts

```diff
@@ -127,40 +127,36 @@ export async function getResolvedFluidRoot(buildRoot = false) {
 	return await realpathAsync(resolvedRoot);
 }
 
+const configName = "repoBuild";
```

Comment:
Why introduce a new config name for "fluid-build"?

Reply:
I reverted these changes back to fluidBuild. If/when we have a new name we can make the changes then.